packages/react/src/Testimonial/Testimonial.tsx

```diff
@@ -89,14 +114,34 @@ const Root = forwardRef(_Root)
  * <Testimonial.Quote>
  */
 type QuoteProps = {
-  children: string
+  children: string | Array<string | React.ReactElement<HTMLElement>>
 } & React.HTMLAttributes<HTMLElement> &
   BaseProps<HTMLElement>
 
 function _Quote({children, className}: QuoteProps, ref) {
+  const childrenArray = useMemo(() => React.Children.toArray(children), [children])
+
+  // TODO: when Firefox supports :has() selector, we should use that instead of JS
+  const getConditionalVariant = useCallback(() => {
+    if (findElementInChildren(children, 'em')) {
+      return 'muted'
+    }
+    return 'default'
+  }, [children])
+
+  const defaultColor = childrenArray.length === 1 ? 'default' : getConditionalVariant()
+
   return (
     <blockquote ref={ref}>
-      <Text className={clsx(styles['Testimonial-quote'], className)}>{children}</Text>
+      <Text
+        className={clsx(
+          styles['Testimonial-quote'],
+          defaultColor === 'muted' && styles['Testimonial-quote--muted'],
```

Comment:
we could just change the variant of the Text component, like we do in `Timeline`

https://github.com/primer/brand/blob/main/packages/react/src/Timeline/Timeline.tsx#L45